modules/eks/actions-runner-controller/variables.tf

```
@@ -178,14 +178,16 @@ variable "runners" {
       scale_down_factor     = optional(string)
     }))
     webhook_driven_scaling_enabled = bool
+    webhook_startup_timeout        = optional(string, null)
     pull_driven_scaling_enabled    = bool
     labels                         = list(string)
-    storage                        = optional(string, "")
+    storage                        = optional(string, false)
+    pvc_enabled                    = optional(string, false)
     resources = object({
       limits = object({
         cpu               = string
         memory            = string
-        ephemeral_storage = optional(string, "")
+        ephemeral_storage = optional(string, false)
```

shouldn't this be 
```suggestion
        ephemeral_storage = optional(string, "false")
```

?

---

It could be, but we would need to update the charts.

we pass these values directly to helm, which then determines if they should be used with something like
`{{- if .Values.pvc_enabled }}` or `{{- if and .Values.dind_enabled .Values.resources.limits.ephemeral_storage }}`
which looks for a boolean `true` or `false`. String would always be `true`

If we change this to `optional(string, "false")`, we would need to instead check for a string with helm instead

---

`optional(string, false)` works for this use case. Although is may be confusing that the default type is a `bool`, whereas the custom type requires a `string`